ci/pipelines/infra-drift-detector-higher-envs.yml

```diff
@@ -159,3 +159,41 @@ jobs:
         trigger: true
       - set_pipeline: infra-drift-detector
         file: infra-drift-detector-pipeline/ci/pipelines/infra-drift-detector-higher-envs.yml
+
+  - name: check-for-zendesk-unsubscribers
+    plan:
+    - get: pay-ci
+    - task: assume-role
+      file: pay-ci/ci/tasks/assume-role.yml
+      params:
+        AWS_ROLE_ARN: arn:aws:iam::((pay_aws_test_account_id)):role/concourse
```

Comment:
This will need to be `pay_aws_staging_account_id` - and you'll need a separate job to check the production unsubscribes (with `pay_aws_production_account_id`).

Reply:
Shall I include the production job in this PR as well?
The staging job went green:
https://pay-cd.deploy.payments.service.gov.uk/teams/pay-deploy/pipelines/infra-drift-detector/jobs/check-for-zendesk-unsubscribers-staging/builds/12